src/common/components/top-navigation/top-navigation.scss

```diff
@@ -69,6 +79,23 @@
         color: $brand-color;
         font-weight: 600;
     }
+
+    .price{
```

Comment:
For mobile, this breaks the top navigation bar. Maybe we just don't display this on mobile?